api/scripts/cms-content-integrity-tests/taskIntegrityTests.ts

```diff
@@ -1,76 +1,87 @@
+import {
+  getIndustryJson,
+  loadAllAddOns,
+  loadAllTasks,
+} from "@businessnjgovnavigator/shared/src/static";
 import { publishSnsMessage } from "@libs/awsSns";
-import { LogWriterType } from "@libs/logWriter";
-import { getIndustryJsonForTest, loadAllAddOns, loadAllTasks } from "@shared/static";
 
-export const checkTaskUsage = async (
-  topicArn: string,
-  stage: string,
-  logger: LogWriterType,
-): Promise<void> => {
-  try {
-    const allTasksFilenames = new Set(loadAllTasks(true).map((task) => task.filename));
+export const checkTaskUsage = async (topicArn: string): Promise<boolean> => {
+  console.log("\n Starting Ceck Task Usage");
 
-    await addOnTaskUsage(allTasksFilenames, topicArn, stage);
-    await industriesTaskUsage(allTasksFilenames, topicArn, stage);
-  } catch (error) {
-    logger.LogError(`Error when running CMS integrity tests: ${error}`);
-  }
+  const allTasksFilenames = new Set(loadAllTasks().map((task) => task.filename));
+
+  const addOnTaskUsageErrors = await addOnTaskUsage(allTasksFilenames, topicArn);
+  const industriesTaskUsageErrors = industriesTaskUsage(allTasksFilenames, topicArn);
```

Comment:
Is this missing an `await`?
```suggestion
  const industriesTaskUsageErrors = await industriesTaskUsage(allTasksFilenames, topicArn);
```